### ISSUE TITLE
Publish ES6 compatible module shape

### ISSUE THREAD

**Issue by bradleyayers** [marijnh/browserkeymap#4]
The current `index.js` uses a UMD style wrapper that has a CommonJS style `module.exports = mod()`, where `mod()` returns a function (rather than an object).

This means that the library isn't able to be imported into an ES6 module unless a hack is used in the packer (e.g. webpack) that translates `import Foo from 'browserkeymap'` to  `require('browserkeymap')` rather than `require('browserkeymap').default`.

This causes a problem for libraries that depend on browserkeymap that want to distribute ES6 modules, as it burdens their consumers with needing their packer configured in this way.

The simplest solution here is to do the export in an ES6 compatible way, i.e.:

```
module.exports = { Keymap: Keymap };
```

or perhaps just adding:

```
Keymap.default = Keymap;
```

**Comment by marijnh** [comment]
I don't know what Webpack is doing here, but auto translating CommonJS modules into ES6 modules isn't something that is generally possible. But if you're going to try it, I would expect assigning to `module.exports` to lead to a `default` export, not breaking the tool. I mean, half the modules on NPM are doing this.

Anyway, note that current ProseMirror versions don't use this library anymore. If you want to submit a patch I'll merge it, but the library doesn't really have much of a future.

**Comment by marijnh** [comment]
Solved via #5
